### PR TITLE
Call script instead of bash in ObsRsync Plugin

### PR DESCRIPTION
**Pull request by andrii-suse**
This way apparmor profile should be more secure

### CODE REVIEW

**Comment by okurz** [comment]
you could control the next OSD deployment yourself to make sure the appropriate script changes are in place. What about o3 though using the daily/nightly deployment?

**Comment by andrii-suse** [comment]
At the moment I don't have permanent solution, other than try to manually manage o3. But at the moment o3 basically waits for this patch to be able use more secure apparmor profile instead of current proposal, which just allows running bash https://github.com/andrii-suse/openqa-trigger-from-obs/pull/1/files

File: lib/OpenQA/WebAPI/Plugin/ObsRsync/Task.pm [hunk 1]
```diff
@@ -52,7 +52,7 @@ sub run {
     return $job->retry({delay => $retry_interval})
       unless my $concurrency_guard = $minion->guard('obs_rsync_run_guard', $lock_timeout, {limit => $concurrency});
 
-    my @cmd = ('bash', "$home/rsync.sh", $project);
+    my @cmd = ("$home/script/rsync.sh", $project);
```
Comment (kraih) [inline lib/OpenQA/WebAPI/Plugin/ObsRsync/Task.pm]: Side note: path building should generally be done by a module that knows how to deal with portability issues, like `Mojo::File`.
```perl
my @cmd = (Mojo::File->new($home, 'script', 'rsync.sh')->to_string, $project);
```

Comment (okurz) [reply lib/OpenQA/WebAPI/Plugin/ObsRsync/Task.pm]: good idea to change this while we are on it. After change we can merge and have it effective on o3 for experimentation and keep it in mind for the next osd deployment to sync with the according changes there.

**Pull request merged**